### ISSUE TITLE
Cannot read properties of undefined (reading 'SINGLE_ELIMINATION')

### ISSUE THREAD

**Issue by larteagar** [Drarig29/brackets-storage#8]
hello, i am trying to use the **brackets-prisma-db** package with **postgres** database and i get the error message described in the title when i try to create a stage
my test code is as follows:

```ts
await this.manager.create.stage({
      name: 'Test stage',
      tournamentId: 1,
      type: 'single_elimination',
      seeding: ['P1','P2','P3','P4','P5', null, null, null],
      settings: {
        seedOrdering: ['inner_outer']
      }
});
```

with an in-memory database there is no problem. any idea what could be happening?

thanks

**Comment by Drarig29** [comment]
Hi! Did you run `npx prisma generate`?

**Comment by larteagar** [comment]
Hi!, we run it. 
We only see that this method succeeds in inserting records in the participant table, after that it shows the error.

**Comment by Drarig29** [comment]
Well, I don't know prisma at all.

https://github.com/Drarig29/brackets-storage/blob/2d86211a6372368f08343b5db014de8ce11f647e/brackets-prisma-db/src/transformers/enum/stage-type.ts#L11

Above is the only place where we are reading `SINGLE_ELIMINATION` so it means that `Prisma.StageType` is undefined for you.

It's defined here:

https://github.com/Drarig29/brackets-storage/blob/2d86211a6372368f08343b5db014de8ce11f647e/brackets-prisma-db/prisma/schema.prisma#L14-L19

cc. @Tandashi

**Comment by larteagar** [comment]
at the beginning I had the same conjecture, I checked it and I see that I have the type is correctly defined in the prism schema and in the migration file and I see it correctly created in the database.
[migration.sql.txt](https://github.com/Drarig29/brackets-storage/files/13060549/migration.sql.txt)

**Comment by larteagar** [comment]
I did the exercise of importing prisma.client

```ts
import * as Prisma from '@prisma/client';
```

and I try to access the enums, I place them in a namespace $Enums

```ts
Prisma.$Enums.StageType.SINGLE_ELIMINATION
````
and not directly in Prisma,  is it a versioning issue ?

**Comment by Tandashi** [comment]
@larteagar Seams like Prisma 5 broke some things. If you need a quick fix try downgrading to Prisma 4.

@Drarig29 Will try to upgrade the `brackets-prisma-db` to work with Prisma 5 as soon as I can.

**Comment by larteagar** [comment]
Yes, I have downgraded to the prisma@4.16.1 version and everything works fine. thanks.

**Comment by Drarig29** [comment]
Hi! This should be fixed with `brackets-prisma-db@2.0.0`
